### PR TITLE
adding dockerization. refactored broken stuff

### LINKED ISSUE #23: containerize samaritan-backend with Docker

### LINKED ISSUE #44: scan_for_known_people_from_db unable to detect faces
method `scan_for_known_people_from_db()`'s encodings from database has an incorrect shape.
method `get_names_and_encodings_from_known()` needs to be changed because of this.
Because of this we can't correctly find `detected_faces`. 

Error trace from failed test from `script_tests` test `test_enncodings_from_db`
```
Error Traceback (most recent call last):
  File "/usr/local/Cellar/python/3.7.4_1/Frameworks/Python.framework/Versions/3.7/lib/python3.7/unittest/case.py", line 59, in testPartExecutor
    yield
  File "/usr/local/Cellar/python/3.7.4_1/Frameworks/Python.framework/Versions/3.7/lib/python3.7/unittest/case.py", line 628, in run
    testMethod()
  File "/Users/anngould/IdeaProjects/samaritan-backend/tests/script_tests.py", line 25, in test_encodings_from_db
    foo = scan_for_known_people_from_db(self.encoded_encoding)
  File "/Users/anngould/IdeaProjects/samaritan-backend/FacialRecog.py", line 112, in scan_for_known_people_from_db
    detected_faces = face_recognition.compare_faces(all_encodings, npy_array)
  File "/Users/anngould/venv/samaritan-backend/lib/python3.7/site-packages/face_recognition/api.py", line 222, in compare_faces
    return list(face_distance(known_face_encodings, face_encoding_to_check) <= tolerance)
  File "/Users/anngould/venv/samaritan-backend/lib/python3.7/site-packages/face_recognition/api.py", line 72, in face_distance
    return np.linalg.norm(face_encodings - face_to_compare, axis=1)
ValueError: operands could not be broadcast together with shapes (16,128) (120,90,3) 
```

### PR DESCRIPTION
**Pull request by gould-ann**
adding dockerization. fixing broken npy arrays atm

### CODE REVIEW

**Comment by katebrune** [comment]
i cleaned up the npy arrays in my pr if ya wanna look, FacialRecog line 170. Is the docker stuff good to go?

File: FacialRecog.py [hunk 1]
```diff
@@ -150,12 +149,16 @@ def get_names_and_encodings_from_known() -> Tuple[list, list]:
...
     all_people = []
 
-    db_get_data = get_all_known("not_api_call")
+    db_get_data = get_known_people("not_api_call")
     for i in db_get_data:
-        foo = i['npy'].strip('][').split(', ')
-        for j in range(len(foo)):
-            foo[j] = float(foo[j])
-        all_encodings.append(foo)
+        # changes STRING of lists^3 (basically a string 3D array) to a list of strings
```
Comment (rgoluch) [inline FacialRecog.py]: what's going on here?

**Review by rgoluch — COMMENTED** [review]
I would hold off on merging in the changes to facialRecog and app.py until we get kate's thing working so we still have the core functionality

File: FacialRecog.py (hunk 1 above)
Comment (gould-ann) [reply FacialRecog.py]: lol Kates thing fixed it we are good

**Comment by gould-ann** [comment]
lets merge this once the other two parts are consolidated

**Comment by gould-ann** [comment]
and just focus on the docker part of it

**Pull request merged**